Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector.py

```diff
@@ -184,6 +131,12 @@ def main():
                     raw_response=events,
                 )
                 return_results(command_results)
+
+            while len(events) > 0:
```

Comment:
no need for batching, you can send it in one request.
```suggestion
            while events:
```